integration/test_batch.py

```diff
@@ -79,6 +82,56 @@ def test_add_data_object(client: weaviate.Client, uuid: Optional[UUID], vector:
     assert has_batch_errors(response) is False, str(response)
 
 
+def test_delete_objects(client: weaviate.Client):
+    with client.batch as batch:
+        batch.add_data_object(data_object={"name": "one"}, class_name="Test")
+        batch.add_data_object(data_object={"name": "two"}, class_name="Test")
+        batch.add_data_object(data_object={"name": "three"}, class_name="Test")
+        batch.add_data_object(data_object={"name": "four"}, class_name="Test")
+        batch.add_data_object(data_object={"name": "five"}, class_name="Test")
+
+    with client.batch as batch:
+        batch.delete_objects(
+            "Test",
+            where={
+                "path": ["name"],
+                "operator": "Equal",
+                "valueText": "one",
+            },
+        )
+    res = client.data_object.get()
+    names = [obj["properties"]["name"] for obj in res["objects"]]
+    assert "one" not in names
+
+    with client.batch as batch:
+        batch.delete_objects(
+            "Test",
+            where={
+                "path": ["name"],
+                "operator": "ContainsAny",
+                "valueTextList": ["two", "three"],
+            },
+        )
+    res = client.data_object.get()
+    names = [obj["properties"]["name"] for obj in res["objects"]]
+    assert "two" not in names
+    assert "three" not in names
+
+    with client.batch as batch:
+        batch.delete_objects(
+            "Test",
+            where={
+                "path": ["name"],
+                "operator": "ContainsAll",
+                "valueTextList": ["four", "five"],
```

Comment:
why wouldn't we just update the documentation to put `valueTextArray` in? They both seem both equally bad?

Reply:
We could certainly do that, it would be a small change on the client and in the docs. I'll make it :D

Reply:
It was implemented in `weaviate/gql/filters.py` as `value<>List` so I kept that convention but that can be undone, not a problem

Reply:
ah wait a minute. Just noticed that the non-array texts are the same as for the GQL filters. So it makes sense to unify that

Reply:
I've unified them all to use `value<>Array` since that's how it's done in other clients :D

Reply:
Ah nice! Just to be sure: This wouldn't break any existing code? Also, is there any documentation that would need to be updated?

Reply:
I introduced the `value<>List` convention in the previous release on Tuesday. Would it be dangerous to go back on this at this stage?